cli/core/pkg/command/context.go

```diff
@@ -240,11 +240,11 @@ func createContextWithKubeconfig() (context *configapi.Context, err error) {
 	}
 
 	context = &configapi.Context{
-		Name:   ctxName,
+		Name:   strings.Trim(ctxName," "),
 		Target: cliv1alpha1.TargetK8s,
 		ClusterOpts: &configapi.ClusterServer{
-			Path:                kubeConfig,
-			Context:             kubeContext,
+			Path:                strings.Trim(kubeConfig," "),
+			Context:             strings.Trim(kubeContext," "),
 			Endpoint:            endpoint,
 			IsManagementCluster: true,
 		},
```

Comment:
Technically, this problem is not limited to this function but will affect (slightly differently) `createContextWithEndpoint()` also.